docker/package/scripts/tezos-accuser-start

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/bash
+set -euo pipefail
+
+accuser="/usr/bin/tezos-accuser-PROTOCOL"
```

Comment:
How does it work? AFAIK in the binary name there is an actual protocol name, not the word PROTOCOL.

Reply:
yes, I `sed` it in the `package_generator.py`

Reply:
[here](https://github.com/serokell/tezos-packaging/pull/67/files#diff-e78056cfeb41547f90c416ed3def3997R148)

Reply:
Alright, but I think these scripts would be move self-contained if you added comments saying that the user of these scripts (even though the only user of these scripts is in this repo) is supposed to replace `PROTOCOL` with the actual protocol. In this case the person who sees these scripts for the first time wouldn't be confused about this `PROTOCOL` thing.

Also, why do you use `sed` instead of environmental variables? The latter seems to be more typical, isn't it?

Reply:
hmm, apparently, you're right, and using env variable is better